frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -447,6 +457,19 @@ const HostDetailsPage = ({
     }
   );
 
+  const {
+    data: hostCertificates,
+    isLoading: isLoadingHostCertificates,
+    isError: isErrorHostCertificates,
+  } = useQuery(
+    ["hostCertificates", host_id],
+    () => hostAPI.getHostCertificates(hostIdFromURL),
+    {
+      ...DEFAULT_USE_QUERY_OPTIONS,
+      enabled: !!hostIdFromURL,
```

Comment:
Should we also disable this based on the host platform for non-Apple devices?

Reply:
yep added.